dist/.log.env.md

```diff
@@ -0,0 +1,93 @@
+
+### Logging handlers
```

Comment:
It doesn't make sense to have this many `#` at the start of the file; it's the equivalent of starting a HTML page with an `h3`. Hierarchy should be:
```
# Logging handlers
## Send logs to a remote logging server
## Send logs to a messaging system
# Log levels
```

Reply:
I'm not sure about this, it was PR from @meker12 https://github.com/magento/ece-tools/pull/709/files
@meker12 Could you please check comment from Ben

Reply:
So I was assuming that this information was being appended to another file -- so I followed the hierarchy that Sasha used, but your comment makes sense @Ben. Need to check the schema file.

Reply:
The content of this file will be a part of automatically generated `.md` file and there is already used `##` and `###`, so it will be strange if after `##` will be added header `#`.
Part of the file
```
## CRON_CONSUMERS_RUNNER

Use this variable to make sure message queues are running after a deployment. By default, the deployment process
overwrites all settings in the env.php file

 cron_run — a boolean value that enables or disables the consumers_runner cron job.

 max_messages — a number specifying the maximum number of messages each consumer must process before terminating.
Although we do not recommend it, you can use 0 to prevent the consumer from terminating.

 consumers — an array of strings specifying which consumer(s) to run. An empty array runs all consumers.

|Attribute|Values|
|---|---|
|Type|array|
|Magento version|\>=2.1.14|
|Stages|deploy|

### Examples

```yaml
  stage:
      deploy:
          CRON_CONSUMERS_RUNNER:
              cron_run: true
              max_messages: 1000
              consumers:
                  - consumer1
                  - consumer2

```

Reply:
Looks like we need to change to `###` -> ` ##`  and `####` -> `###`

Reply:
so I think Ben is correct that the levels are not quite correct.

I believe that the variable descriptions need to be at the H3 level, and the titles like Logging handlers will be at the h2 level.

This is setting the markdown up to be inserted into a template for a doc page-- 

![image](https://user-images.githubusercontent.com/6391769/78281502-638e7c00-74e0-11ea-9a81-6f775228be15.png)